dyninstAPI/src/image.C

```diff
@@ -61,11 +61,6 @@
 #include <sys/time.h>
 #endif
 
-#if defined( cap_dwarf )
```

Comment:
In ./dyninstAPI/src/BPatch_collections.C there is also a #if defined(cap_dwarf)
I'm not sure if it is related, but it seems like the guard there can be removed as well.

Reply:
based on my understanding of the logic of that piece of  code.

Reply:
That check can probably be removed, but it's beyond the scope of what I want to change here.